### ISSUE TITLE
Prev/Next change in diff viewer(when file is openned in working tree) is not announce the line and content when changed

### ISSUE THREAD

**Issue by ilhom0258** [microsoft/vscode#239709]
- VS Code Version: latest
- OS Version: Windows 11

Issue: Shift + Alt + F5 & Alt + F5 does not announce change for accessibility tools (NVDA, JAWS)

Steps to Reproduce:

1. Open diff viewer
2. Click Alt + F5 or Shift + Alt + F5
3. Notice that change of line is not being announced

**Comment by meganrogge** [comment]
Hi, are you on insider's with screen reader mode enabled?
